editors/mediawiki.el/Portfile

```diff
@@ -1,34 +1,36 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
 PortSystem 1.0
+PortGroup github 1.0
 
+github.setup            hexmode mediawiki-el 2.3.1
 name                    mediawiki.el
-version                 2.2.4.1
-categories              editors 
+categories              editors
 license                 GPL-3
 maintainers             {easieste @easye} openmaintainer
 description             An Emacs mode for editing MediaWiki content
 long_description        ${description}
 homepage                http://www.emacswiki.org/emacs/MediaWikiMode
 platforms               darwin
-distname                mediawiki.el
-master_sites            https://launchpadlibrarian.net/127264410/
 
-checksums           rmd160  35b75da2e09931bfd0b4b0746a599c6fac60a024 \
-                    sha256  9ebc5c1156af54bb533eae7273ef567c2f0cadbe870be9b4759ea70a214ff351
-extract.suffix
+checksums           rmd160  4f011a2efac2f406958361ef8063d703658a9f57 \
+                    sha256  80fb1d3ba5d1f66388b0758cee70e2a27299f4a5244b6d13fd29466851c020ab \
+                    size    36627
```

Comment:
can you please align this the same as the rest of the Portfile

Reply:
ok, done in ba25bf1